### PR TITLE
Add argparse

### PR DESCRIPTION
**Pull request by jakirkham**
Generated with `conda skeleton pypi`. Basically for backcompat. Might try to build once and otherwise it will just be disabled.

### CODE REVIEW

**Comment by jakirkham** [comment]
Looks like this will also run into issue ( https://github.com/conda-forge/conda-smithy/issues/257 ).

**Comment by jakirkham** [comment]
Addressed in the last commit.

**Comment by jakirkham** [comment]
Re-investigated this and it looks like the original skip restriction is fine. The aforementioned problems seems to be restricted to dependencies.

File: recipes/argparse/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - argparse
+
+about:
+  home: https://github.com/ThomasWaldmann/argparse/
+  license: Python Software Foundation License
```
Comment (jakirkham) [inline recipes/argparse/meta.yaml]: Need version.

**Comment by msarahan** [comment]
Please invite original author to be maintainer if they'd like - though I'm sure sue were going to, anyway.

**Pull request merged**